NgocMaiDang_democypress/cypress/e2e/addbook_spec.cy.js

```diff
@@ -0,0 +1,39 @@
+Cypress.on('uncaught:exception', (err, runnable) => {
+    // returning false here prevents Cypress from
+    // failing the test
+    return false
+});
+  
+describe('Add book with name "Git Pocket Guide" to my collection and verify it in my profile', () => {
+    it('Should add the book to the collection and verify it in the profile', () => {
+  
+      //Visit the login page and login
+      cy.visit('https://demoqa.com/login');
```

Comment:
Login and go to book store page should be place in `beforeEach()` because they are pre-conditions